src/shared/utilities/render-html.ts

```diff
@@ -15,6 +15,14 @@ interface DomElement {
 // eslint-disable-next-line @typescript-eslint/ban-types
 export type ParseHTMLReplacer = (domNode: DomElement) => JSX.Element | object | void | undefined | null | false;
 
+DOMPurify.setConfig({ ADD_ATTR: ['target'] });
 export function renderHTML(html: string, replace?: ParseHTMLReplacer) {
-  return parse(DOMPurify.sanitize(html || ""), { replace });
+  let newHtml;
+  if (html.includes("<a")) {
+    const position = html.indexOf("<a") + 2;
+    newHtml = [html.slice(0, position), " target=\"_blank\"", html.slice(position)].join("");
+  } else {
+    newHtml = html;
+  }
```

Comment:
This kind of string manipulation is brittle and error-prone. It would be better to enforce this at the node level rather than via string manipulation. There is probably a way to accomplish this via the `replace` option of the parser but in this case I think hooking into `DOMPurify` via the `afterSanitizeAttributes` hook may make more sense. I came across an example that looked like this:
```javascript
DOMPurify.addHook("afterSanitizeAttributes", function(node) {
  // Fix elements with `target` attribute:
  // - allow only `target="_blank"
  // - add `rel="noopener noreferrer"` to prevent https://www.owasp.org/index.php/Reverse_Tabnabbing

  const target = node.getAttribute("target");
  if (isString(target) && target.toLowerCase() === "_blank") {
    node.setAttribute("rel", "noopener noreferrer");
  } else {
    node.removeAttribute("target");
  }
});
```
In this case I think it would look something more like (note that this is untested code, so you'll presumably have to futz with it to get it right):
```javascript
DOMPurify.addHook("afterSanitizeAttributes", function(node) {
  // Fix `a` elements:
  // - add `target="_blank"
  // - add `rel="noopener noreferrer"` to prevent https://www.owasp.org/index.php/Reverse_Tabnabbing
  if (node.tagName.toLowercase() === "a") {
    node.setAttribute("target", "_blank");
    node.setAttribute("rel", "noopener noreferrer");
  }
});
```
Separately, I've thought about publishing `renderHtml()` as a reusable npm module, in which case this kind of configuration should be client-configurable rather than being built into the function, but that possibility can be left for another day.